lua/orgmode/objects/date.lua

```diff
@@ -194,8 +229,8 @@ end
 ---@param data? table
 ---@return OrgDate
 local function today(data)
-  local date = os.date('*t', os.time()) --[[@as osdate]]
-  local opts = vim.tbl_deep_extend('force', date, data or {})
+  local date = os_date(os.time())
```

Comment:
Should we use `os_time` here?

```suggestion
  local date = os_date(os_time())
```

Reply:
No, we want to get the current time. The argument of `os.time` is optional. We use `os_time` to parse a date and get rid of the type warnings. Because we feed the result of `os.time` in our warning-swallowing `os_date` this line is fine as it is.